SteamKit2/SteamKit2/Util/CryptoHelper.cs

```diff
@@ -538,5 +538,15 @@ public static byte[] GenerateRandomBlock( int size )
             }
         }
 
+        private static int ReadAll( Stream stream, byte[] buffer )

```

Comment:
I'd be tempted to make this an extension method, then we can reuse it in other places like ZipUtil below quite easily.

Reply:
I thought about it, I have no preference one way or the other. Which one you prefer?

Reply:
Either extension method, or helper method in Utils that ZipUtil also uses.
I’m not fussed which way you choose but I’d rather the code not be duplicated.